authorization/2.0/_index.md

```diff
@@ -10,17 +10,17 @@ these permissions.
 
 * [Jakarta Authorization 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.jacc/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Authorization 2.0 Specification Document](./authorization-spec-2.0-SNAPSHOT.pdf) (PDF)
-* [Jakarta Authorization 2.0 Specification Document](./authorization-spec-2.0-SNAPSHOT.html) (HTML)
+* [Jakarta Authorization 2.0 Specification Document](./authorization-spec-2.0.pdf) (PDF)
+* [Jakarta Authorization 2.0 Specification Document](./authorization-spec-2.0.html) (HTML)
 * [Jakarta Authorization 2.0 Javadoc](./apidocs)
-* [Jakarta Authorization 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Authorization 2.0 TCK]()(https://download.eclipse.org/jakartaee/authorization/2.0/jakarta-authorization-tck-2.0.0.zip)([sig](https://download.eclipse.org/jakartaee/authorization/2.0/jakarta-authorization-tck-2.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/authorization/2.0/jakarta-servlet-tck-2.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.authorization:jakarta.authorization-api:jar:2.0.0-RC1](https://search.maven.org/artifact/jakarta.authorization/jakarta.authorization-api/2.0.0-RC1/jar)
+  * [jakarta.authorization:jakarta.authorization-api:jar:2.0.0](https://search.maven.org/artifact/jakarta.authorization/jakarta.authorization-api/2.0.0/jar)
 
 
 # Compatible Implementations
 
-* [Eclipse Glassfish 6.0.0-M1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads)
+* [Eclipse Glassfish 6.0.0-M2-servlet5](https://github.com/eclipse-ee4j/glassfish/releases/tag/6.0.0-M2-servlet5)
 
```

Comment:
The template is wrong... :-)  That needs to be corrected.  We moved the compatible implementation section up above the ballots so that it doesn't get lost in the weeds...